.github/workflows/ac-ansible-test-sanity.yml

```diff
@@ -0,0 +1,75 @@
+name: AC Ansible sanity
+
+on:
+  pull_request:
+    branches:
+      - dev
+      - staging*
+    paths-ignore:
+      - '**.tar.gz'
+      - 'pycache/**'
+      - '.ansible-lint'
+      - 'cache/**'
+      - '.DS_Store'
+      - '.git/**'
+      - '.github/**'
+      - '.gitignore'
+      - '.python-version'
+      - '.pytest_cache/**'
+      - '.vscode/**'
+      - 'Jenkinsfile'
+      - 'ac'
+      - 'ansible.cfg'
+      - 'changelogs/**'
+      - 'collections/**'
+      - 'docs/**'
+      - 'scripts/**'
+      - 'test_config.yml'
+      - 'tests/*.ini'
+      - 'tests/*.py'
+      - 'tests/.pytest_cache'
+      - 'tests/pycache'
+      - 'tests/functional'
+      - 'tests/helpers'
+      - 'tests/requirements.txt'
+      - 'tests/unit'
+      - 'tests/sanity/ignore-2.9'
```

Comment:
i would suggest either tests/sanity/ignore-* or adding 2.14 (I think the * would be easier to maintain)